### DISCUSSION TITLE
mmm what the -r actually do !!

### DISCUSSION THREAD

**Discussion by c4ir0** [projectdiscovery/shuffledns#150] [Q&A] [answered]
```
┌──(kali㉿kali)-[~]
└─$ shuffledns -d subs -list words.txt -r resolver.txt

       __        ________        __       
  ___ / /  __ __/ _/ _/ /__  ___/ /__ ___
 (_-</ _ \/ // / _/ _/ / -_)/ _  / _ \(_-<
/___/_//_/\_,_/_//_//_/\__/ \_,_/_//_/___/ v1.0.8

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[FTL] Program exiting: resolver file doesn't exists

```

**I just wanna know what should the `resolver.txt` file must contain ?!?**
**and thx as always for being big helpers with the many times I asked in and got the solution ❤**

**Comment by ehsandeep** [comment, marked as answer]
@c4ir0 please do not tag individuals, opening a discussion should be sufficient.

as the help menu suggest, in order to perform brute force you need to provide dns resolver list, you can get it from https://github.com/trickest/resolvers

```yaml
   -r, -resolver string    File containing list of resolvers for enumeration
```

**Reply by c4ir0** [reply]
srry for mentioning and thx for help, will take care of the details next time
